engine/gamesys/src/gamesys/components/comp_model.cpp

```diff
@@ -364,6 +364,10 @@ namespace dmGameSystem
     static void CompModelPoseCallback(void* user_data1, void* user_data2)
     {
         ModelComponent* component = (ModelComponent*)user_data1;
+        if (!component->m_Resource->m_Model->m_CreateGoBones)
+        {
+            return;
+        }
```

Comment:
If we don't want this callback, I think it's probably better to not register it?
This will be called each update, and we do 3 pointer dereferences here.